acceptance_tests/pyramid_ogcapi_test_app/ogcapi/__init__.py

```diff
@@ -30,18 +31,14 @@ def landing_page(self, pyramid_request: pyramid.request.Request, request: Any) -
         The landing page provides links to the API definition, the conformance
         statements and to the feature collections in this dataset.
         """
-        del pyramid_request, request  # Unused
+        del request  # Unused
+        spec = pyramid_request.registry.settings["ogcapi_features"]["spec"]
```

Comment:
From me, this should be in the `link` function, and the `"ogcapi_features"]` should be an argument.